src/olympia/lib/crypto/signing.py

```diff
@@ -111,8 +111,8 @@ def call_signing(file_obj):
         'keyid': conf['signer'],
         'options': {
             'id': get_id(file_obj.version.addon),
-            # Add-on variant A params (PKCS7 SHA256 and COSE ES256) work in
-            # Fx >57 which is now the minimum version of Fx supported for signing.
+            # Add-ons dual-signed with PKCS7 SHA256 and COSE ES256 work in
+            # Fx > 58 which is now the minimum version of Fx supported for signing.
```

Comment:
```suggestion
            # Fx >= 58 which is now the minimum version of Fx supported for signing.
```

right ?

Reply:
Our checks are all `>=` - when we set `DEFAULT_WEBEXT_MIN_VERSION` to `58.0` the add-ons will work in `58.0` or greater.

The comment before was a bit confusing though. I'll suggest a new variant